packages/x-components/src/x-modules/search/store/module.ts

```diff
@@ -81,9 +82,7 @@ export const searchXStoreModule: SearchXStoreModule = {
     setPage(state, page) {
       state.page = page;
     },
-    setPageSize(state, pageSize) {
-      state.config.pageSize = pageSize;
-    },
```

Comment:
Removing this mutation (or renaming it) is a breaking change. You have to document it in the PR description and [in the commit message](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer)